### ISSUE TITLE
SyntaxError: invalid syntax Exited with code 1

### ISSUE THREAD

**Issue by brianhhq** [uptick/pymyob#14]
from myob.credentials import PartnerCredentials
  File "/home/circleci/repo/venv/lib/python3.4/site-packages/myob/__init__.py", line 1, in <module>
    from .api import Myob  # noqa
  File "/home/circleci/repo/venv/lib/python3.4/site-packages/myob/api.py", line 2, in <module>
    from .managers import Manager
  File "/home/circleci/repo/venv/lib/python3.4/site-packages/myob/managers.py", line 70
    **request_kwargs_raw,

**Comment by jarekwg** [comment]
Thanks for the report, but I can't seem to reproduce this.
```
$ python
Python 3.6.1 (v3.6.1:69c0db5050, Mar 21 2017, 01:21:04)
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import myob
>>> myob.VERSION
(0, 4, 4)
>>> from myob.credentials import PartnerCredentials
>>>
```
Which version of pymyob are you using?
Could you please try to reproduce this locally and provide steps, or provide a fuller stack trace?

Cheers,

**Comment by jarekwg** [comment]
Dw, found it. It's a weird python 3.4 thing with trailing comma after kwargs...
Pushing up fix presently..

**Comment by jarekwg** [comment]
Fixed in 241d6ad20e248eaa950a2e21af9d4d9d32c07e96.
Though I recommend upgrading to Python3.6. f-strings are the bees' knees.
